src/containers/Sidebar/NavbarBrand.js

```diff
@@ -1,42 +1,63 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import { connect, useDispatch } from 'react-redux';
 
 import { Link } from 'react-router-dom';
+import { SET_SMALL_SIDEBAR } from "../../actions";
 
-const NavbarBrand = ({ title, brand_image, isSidebarCollapsed }) => {
+const NavbarBrand = ({ title, brand_image, isSmallSidebarOpen }) => {
 	const href = brand_image.href ?? "/";
+	const dispatch = useDispatch();
+
+	const onCollapse = () => {
+		dispatch({
+			type: SET_SMALL_SIDEBAR,
+			isSmallSidebarOpen: !isSmallSidebarOpen
+		});
+	}
 
 	if (href.includes("http")) {
 		return (
-			<div className="sidebar-brand-image">
-				<a
-					href={href}
-					target="_blank"
-					rel="noopener noreferrer"
-				>
+			<>
+				<div className="header-sidebar-toggler" onClick={onCollapse}>
+					<i className="cil-menu"></i>
+				</div>
+				<div className="sidebar-brand-image">
+					<a
+						href={href}
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						<img
+							// src={isSidebarCollapsed ? brand_image.minimized : brand_image.full}
+							src={brand_image.full}
+							alt={title}
+							width="50"
+							height="50"
+							className="minimized-image"
+						/>
+					</a>
+				</div>
+			</>
+		);
+	}
+	return (
+		<>
+			<div className="header-sidebar-toggler" onClick={onCollapse}>
+				<i className="cil-menu"></i>
+			</div>
+			<div className={`sidebar-brand-image`}>
+				<Link to={href}>
 					<img
-						src={isSidebarCollapsed ? brand_image.minimized : brand_image.full}
+						// src={isSidebarCollapsed ? brand_image.minimized : brand_image.full}
+						src={brand_image.full}
```

Comment:
@Aringoaway I dont think this is correct, we want to have minimized logo when sidebar is just collapsed in large screens.